DOCSTYLE.md

```diff
@@ -0,0 +1,45 @@
+# Documentation style guidance
+
+We've defined some basic patterns to bring our documentation towards a more consistent, readable style.
+
+- We are committed to inclusive communication
+- We use British English spelling (e.g., "organisation" not "organization", "centre" not "center")
+- We prefer "-ise" over "-ize" endings
+- We spell out numbers one through ten, use numerals for 11+
```

Comment:
why?

Reply:
This is an organisational communications preference and guidance we're adopting as a team. Happy to expand this file to make this more clear.

Reply:
That said I have removed the suggestion that numbers be spelled out in this manner as I can see that it is less applicable in tech documentation.